test/cljs_time/coerce_test.cljs

```diff
@@ -81,7 +81,9 @@
   (is (= 893462400 (to-epoch (js/Date. 893462400000))))
   (is (= (long 0) (to-epoch 0)))
   (is (= 893462400 (to-epoch 893462400000)))
-  (is (= 893462400 (to-epoch "1998-04-25T00:00:00.000Z"))))
+  (is (= 893462400 (to-epoch "1998-04-25T00:00:00.000Z")))
+  (testing "returns an int"
```

Comment:
I'd put the testing block around both, so it's clear that you're comparing and contrasting their behaviour. e.g. `(testing "times with fractions of a second are returned as ints)`.